src/Dapr.Extensions.Configuration/Dapr.Extensions.Configuration.csproj

```diff
@@ -13,10 +14,6 @@
     <ProjectReference Include="..\Dapr.Client\Dapr.Client.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Compile Include="..\Dapr.Client\ArgumentVerifier.cs" />
-  </ItemGroup>
```

Comment:
Instead of copying this into the project can you create `..\Shared\ArgumentVerifierNullable.cs`? Then we can limit the amount of duplication to N == 2.